src/plugins/inputs/otr.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function(network, chan, cmd, args) {
+	if (cmd !== "otr") {
+		return;
+	}
+	if (args.length === 0 || args[0] === "") {
+		return;
+	}
+	var subCmd = args[0];
+	var client = this;
+	var irc = network.irc;
+	var otrStore = client.otrStore;
+	var session = otrStore.getSession(chan.title, network);
+
+	if (subCmd === "init") {
```

Comment:
I noticed your version uses a `switch`... I will when we have more to check against.

This is very personal and not motivated, but I'm not too much into switches, but I force myself to use them when it makes sense. Oh and [Scala's pattern matching](http://docs.scala-lang.org/tutorials/tour/pattern-matching.html), which is like a switch under steroids, is like my current favorite thing so, you know, I'm not even consistent with myself :-)